nomos-services/storage/Cargo.toml

```diff
@@ -8,24 +8,27 @@ edition = "2021"
 [dependencies]
 async-trait = "0.1"
 futures = "0.3"
-tokio = { version = "1", features = ["sync"] }
+tokio = { version = "1.41.1", features = ["sync"] }
```

Comment:
Why fix the version? We recently changed all deps to use just latest 1.

Reply:
I thought I needed it when I was fixing some dep incompatibility, but it's not anymore. Removing.